sdk/python/kubeflow/training/api/training_client_test.py

```diff
@@ -589,16 +612,85 @@ def test_wait_for_job_conditions(training_client, test_name, kwargs, expected_ou
 ]
 
 
+test_data_get_job = [
```

Comment:
Can you move this variable here: 
https://github.com/kubeflow/training-operator/blob/2e2e138d621e0f08357253803fb5dcc1db86d91f/sdk/python/kubeflow/training/api/training_client_test.py#L466
So we can be consistent with other test data.